<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PLSLiveInfoYoutube</class>
 <widget class="QWidget" name="PLSLiveInfoYoutube">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LiveInfo.liveinformation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="PLSLiveInfoDualWidget" name="dualWidget" native="true"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1251</width>
        <height>515</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="QWidget" name="thumWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="PLSSelectImageButton" name="thumbnailButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter|Qt::AlignTop">
           <widget class="QLabel" name="thumDetailLabel">
            <property name="text">
             <string>2MB text only</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="midwidget" native="true">
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="rowWrapPolicy">
           <enum>QFormLayout::DontWrapRows</enum>
          </property>
          <property name="labelAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>24</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>14</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="scheduleLabel">
            <property name="text">
             <string>LiveInfo.Youtube.scheduled.title</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="PLSScheduleCombox" name="sPushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="liveTitleLabel">
            <property name="text">
             <string>LiveInfo.base.Title</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="PLSLineEdit" name="lineEditTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="publicLabel">
            <property name="text">
             <string>LiveInfo.Youtube.privacy</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="PLSComboBox" name="comboBoxPublic">
            <property name="editable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="categoryLabel">
            <property name="text">
             <string>LiveInfo.Base.Category</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="PLSComboBox" name="comboBoxCategory">
            <property name="editable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="describeLabel">
            <property name="text">
             <string>LiveInfo.Base.Description</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="PLSTextEdit" name="textEditDescribe">
            <property name="acceptRichText">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="latencyTitle">
            <property name="text">
             <string>LiveInfo.latency.Title</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QWidget" name="latencyRadioWidget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>15</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="PLSRadioButton" name="radioButton_normal" native="true">
               <property name="text" stdset="0">
                <string>LiveInfo.latency.normal</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="PLSRadioButton" name="radioButton_low" native="true">
               <property name="text" stdset="0">
                <string>LiveInfo.latency.low</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="PLSRadioButton" name="radioButton_ultraLow" native="true">
               <property name="text" stdset="0">
                <string>LiveInfo.latency.ultra.low</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="ultraLowTipLabel">
               <property name="text">
                <string>LiveInfo.latency.ultra.low.tip</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="kidsLabel">
            <property name="text">
             <string/>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QWidget" name="kidsRadioWidget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="PLSRadioButton" name="kidsRadioButton" native="true">
               <property name="text" stdset="0">
                <string>LiveInfo.Kids.ForKids</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="PLSRadioButton" name="notKidsRadioButton" native="true">
               <property name="text" stdset="0">
                <string>LiveInfo.Kids.Not.ForKids</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="PLSRadioButton" name="hiddenRadioButton" native="true">
               <property name="text" stdset="0">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>24</number>
         </property>
         <property name="rightMargin">
          <number>14</number>
         </property>
         <property name="bottomMargin">
          <number>39</number>
         </property>
         <item>
          <widget class="QLabel" name="hlsTipLabel">
           <property name="font">
            <font>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>youtube.liveinfo.hls.tip</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="lineView" native="true"/>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QWidget" name="bottomButtonWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="useFor" stdset="0">
            <string notr="true">Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rehearsalButton">
           <property name="text">
            <string>Rehearsal</string>
           </property>
           <property name="useFor" stdset="0">
            <string notr="true">Rehearsal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
           <property name="useFor" stdset="0">
            <string notr="true">Ok</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PLSLineEdit</class>
   <extends>QLineEdit</extends>
   <header>PLSEdit.h</header>
  </customwidget>
  <customwidget>
   <class>PLSScheduleCombox</class>
   <extends>QPushButton</extends>
   <header>PLSScheduleCombox.h</header>
  </customwidget>
  <customwidget>
   <class>PLSSelectImageButton</class>
   <extends>QLabel</extends>
   <header>PLSSelectImageButton.h</header>
  </customwidget>
  <customwidget>
   <class>PLSComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">PLSComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>PLSRadioButton</class>
   <extends>QWidget</extends>
   <header location="global">PLSRadioButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PLSTextEdit</class>
   <extends>QTextEdit</extends>
   <header>PLSEdit.h</header>
  </customwidget>
  <customwidget>
   <class>PLSLiveInfoDualWidget</class>
   <extends>QWidget</extends>
   <header>PLSLiveInfoBase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
